src/widgets/hits-per-page/hits-per-page.js

```diff
@@ -1,7 +1,6 @@
 import React, { render, unmountComponentAtNode } from 'preact-compat';
 import cx from 'classnames';
```

Comment:
is this import still used?

Reply:
shoudn't the linter detect it otherwise?

Reply:
It is still used to compute the suit CSS classes below.